domains/baumistlustig.json

```
@@ -5,6 +5,6 @@
     },
 
     "record": {
-        "URL": "https://baumistlustig.eu"
+        "URL": "hoellwerth.eu"
```

Invalid URL. That is still a URL record.
```suggestion
        "CNAME": "hoellwerth.eu"
```